yarn.lock

```diff
@@ -37304,15 +37304,6 @@ shelljs@0.8.5:
     interpret "^1.0.0"
     rechoir "^0.6.2"
 
-shelljs@^0.8.4:
```

Comment:
Why did this get updated?

Reply:
Updated after a `yarn install`, looks to be appropriate?

Reply:
I think so - I've seen this as well, yarn.lock seems out of sync - someone added a dep but didn't commit the lockfile.